packages/patternfly-3/patternfly-react-extensions/src/components/FilterSidePanel/FilterSidePanelCategoryItem.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import { Checkbox, excludeKeys } from 'patternfly-react';
+
+const FilterSidePanelCategoryItem = ({ children, className, icon, count, ...props }) => {
+  const classes = classNames('filter-panel-pf-category-item', className);
+
+  return (
+    <div className={classes}>
+      <Checkbox {...props}>
+        {icon && <span className="item-icon">{icon}</span>}
+        {children}
+        {Number.isInteger(count) && <span className="item-count">{`(${count})`}</span>}
```

Comment:
Inside the span can just have `{count}`

Reply:
we want the count inside parens, like `(3)`